### DISCUSSION TITLE
Nulleinspeidung

### DISCUSSION THREAD

**Discussion by RobertPannek** [tbnobody/OpenDTU#834] [Ideas]
Hallo, 

Erstmal ein riesen Dank für das Projekt OPENDTU und die damit verbundene Arbeit.
Ich kannhier wirklich nur meinen Respekt zollen !!!! 👍 👍 👍 

Zu meiner Frage:

Ist es in Planung eine Nulleinspeisereglung umzusetzten mit Hilfe einers Strommessers bspw. einer DTSU oder dem Lesekopf für den Stromzähler von Hitchi ?? 

Danke vorab für eine Info

**Comment by helgeerbe** [comment]
Du kannst ja mal hier schauen.
https://github.com/helgeerbe/OpenDTU-OnBattery
Ein Dynamic Power Limiter ist implementiert. Funktioniert auch ohne angeschlossene Batterie. Allerdings wird aktuell nur ein Wechselrichter zur Regelung der Nulleinspeisung unterstützt. Der Haupteinsatszweck ist eben, dass die Energie der Batterie auch wirklich im Haushalt verbraucht wird.
